### ISSUE TITLE
Support for multi store setup

### ISSUE THREAD

**Issue by VeerleDoyen** [dpdconnect/magento2-shipping#51]
Do you plan to support multi store setup?

**Comment by dpdplugin** [comment]
Hi @VeerleDoyen ,

Multistore should be supported, it is however true that you mostly get 1 set of DPD credentials you need to use for every store.
What problems are you encountering exactly?

**Comment by VeerleDoyen** [comment]
Hi,
After installing the page Stores - Configuration - Sales - Shipping Methods
gave an empty page and the checkout page of our website is not working
anymore.
I rebuilt everything already but it stays. I didn't find an exception in
the log files.
Any ideas?
In your documentation
https://doc.dpd.be/sites/default/files/pluginpdf/DPD%20Magento2%20-%20Shopowner%20manual.pdf
under point 2 there was mentioned "This plug in does not support a multi
store setup.". So that's why I proposed that that was the problem.

Kind regards,
Veerle

[image: image.png]

Op di 22 mrt. 2022 om 10:28 schreef DPD Support for E-Commerce Benelux <
***@***.***>:
